python/cuda_parallel/tests/test_reduce.py

```diff
@@ -557,13 +548,13 @@ def add_op(x, y):
         d_out=d_out,
         num_items=d_in.size,
         h_init=h_init,
-        stream=stream_wrapper,
+        stream=cuda_stream,
     )
-    with stream:
+    with cp_stream:
         d_temp_storage = cp.empty(temp_storage_size, dtype=np.uint8)
```

Comment:
Is it necessary to set a different default stream for making an allocation? Is it because we need allocation is made using stream-ordered allocator?

Reply:
Yes, see prior discussion here: https://github.com/NVIDIA/cccl/pull/3348#discussion_r1919434358